presto-main/src/main/java/com/facebook/presto/server/security/RoleType.java

```diff
@@ -0,0 +1,10 @@
+package com.facebook.presto.server.security;
+
+public final class RoleType
```

Comment:
I believe that we should move this class to `presto-common` package. @arhimondr - thoughts?

@arhimondr - do you think we

Reply:
Package `com.facebook.presto.common.type` seems like a fair option.

Reply:
If we really want to have authz for Presto proxy, then we should move it to `presto-common`. Importing `presto-main` to `presto-proxy` is definitely bad idea.